<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vClient</class>
 <widget class="QWidget" name="vClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vClient</string>
  </property>
  <widget class="QLabel" name="title_lb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>可视化Demo客户端</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>841</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <property name="spacing">
     <number>60</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="modelPath_lb">
          <property name="text">
           <string>模型路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="modelPath_le"/>
        </item>
        <item>
         <widget class="QPushButton" name="modelPath_btn">
          <property name="text">
           <string>选择模型</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="imgPath_lb">
          <property name="text">
           <string>图片路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="imgPath_le"/>
        </item>
        <item>
         <widget class="QPushButton" name="imgPath_btn">
          <property name="text">
           <string>选择图片</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="ip_lb">
          <property name="text">
           <string>IP地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ip_le">
          <property name="placeholderText">
           <string>请输入IP地址</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="port_lb">
          <property name="text">
           <string>端口号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="port_le">
          <property name="placeholderText">
           <string>请输入端口号</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="connect_btn">
          <property name="text">
           <string>连接</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="status_lb">
          <property name="text">
           <string>状态：未连接</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="tips_lb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="imgPreview_lb">
        <property name="text">
         <string>图片预览：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="imgShow_lb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="result_tbw"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="exit_btn">
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="classify_btn">
          <property name="text">
           <string>识别</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
